ramls/userSummary.json

```diff
@@ -15,7 +15,21 @@
     "userId": {
       "description": "Unique ID of the user",
       "type": "string",
-      "$ref": "raml-util/schemas/uuid.schema"
+      "$ref": "raml-util/schemas/uuid.schema",
+      "x-fqm-joins-to-raw": [
+        {
+          "targetId": "bb058933-cd06-4539-bd3a-6f248ff98ee2",
+          "targetField": "id",
+          "type": "equality-cast-uuid",
+          "direction": "left"
+        },
+        {
+          "targetId": "f2615ea6-450b-425d-804d-6a495afd9308",
+          "targetField": "id",
+          "type": "equality-cast-uuid",
+          "direction": "left"
+        }
+      ]
```

Comment:
In the future, these should be changed to point simply to `mod-users`'s entities by their non-raw name (we support a schema like `{ targetModule: 'mod-users', targetEntity: 'user', ... }`), however, this will only work once mod-users migrates to auto-generated entity types.